pylint/lint/pylinter.py

```diff
@@ -223,7 +223,7 @@ def make_options():
                     "type": "regexp_csv",
                     "metavar": "<pattern>[,<pattern>...]",
                     "dest": "black_list_re",
-                    "default": (),
+                    "default": (r"^\.#",),
```

Comment:
Should we add this to our own `pylintrc`? The `ignore-paths` option is currently not included.

Reply:
Probably something to do in the upgrade of our pylintrc so it matches the one generated by pylint more closely ?

Reply:
Okay!